server/conn_stmt.go

```diff
@@ -295,6 +299,7 @@ const (
 
 func (cc *clientConn) handleStmtFetch(ctx context.Context, data []byte) (err error) {
 	cc.ctx.GetSessionVars().StartTime = time.Now()
+	cc.ctx.GetSessionVars().EndAlloc()
```

Comment:
Fetch is used by the prepared/exec protocol?

Reply:
used by fetch request（Maybe java api or c api）